migrate/repo_checks.py

```diff
@@ -0,0 +1,364 @@
+"""
+Run checks Against Repos and correct them if they're missing something.
+
+"""
+from itertools import chain
+from pprint import pformat
+
+import click
+import requests
+from fastcore.net import HTTP4xxClientError, HTTP5xxServerError, HTTP404NotFoundError
+from ghapi.all import GhApi, paged
+
+
+class Check:
```

Comment:
For all of these, I'm assuming that this is something we're going to be building on in the future. If it's likely to only be used once or twice, it's probably not worth the effort to refactor around.